### PR TITLE
Add Leveraged-Authorization Documentation

### PR DESCRIPTION
**Pull request by Gabeblis**
This PR aims to add the missing leveraged authorization documentation to support PRs [#913](https://github.com/GSA/fedramp-automation/pull/913), [#921](https://github.com/GSA/fedramp-automation/pull/921), and [#927](https://github.com/GSA/fedramp-automation/pull/927).

### CODE REVIEW

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 1]
```diff
@@ -915,10 +915,10 @@ Each system must define at least two data centers. There must be exactly one pri
 ---
 ## Leveraged FedRAMP-authorized Services
 
-If this system is leveraging the authorization of one or more systems, such as a SaaS running on an IaaS, each leveraged system must be represented within the system-implementation assembly. There must be one leveraged-authorization assembly and one matching component assembly for each leveraged authorization.
+If this system is leveraging the authorization of one or more systems, such as a SaaS running on an IaaS, each leveraged system must be represented within the system-implementation assembly. There must be one leveraged-authorization assembly and one matching component assembly for each leveraged authorization. A leveraged authorization must define a FIPS-199 impact level (low, moderate, or high) that aligns with or exceeds the security sensitivity level of the leveraging system.
```
[the remark this answers is not in the document]
Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Thanks. I committed your suggestion. 👍

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 2]
```diff
@@ -960,8 +963,20 @@ While a leveraged system has no need to represent content here, its SSP must inc
...
-    <component uuid="uuid-of-leveraged-system" type="system">
+    <!-- CSO name & service description -->
+    <component uuid="00000000-0000-0000-0000-000000000000" type="system">
+      <prop name="leveraged-authorization-uuid" value="11111111-3333-5555-0000-000000000001"/>
+      <prop name="implementation-point" value="external"/>
+      <prop ns="http://fedramp.gov/ns/oscal" name="nature-of-agreement" value="sla"/>
+      <prop ns="http://fedramp.gov/ns/oscal" name="authentication-method" value="yes">
+        <remarks>
+          <p>If 'yes', describe the authentication method.</p>
+          <p>If 'no', explain why no authentication is used.</p>
```
Comment (DimitriZhurkin) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:974]: Very minor, I'd change it slightly:
```suggestion
          <p>If 'no', explain why authentication is not used.</p>
```

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 3]
```diff
@@ -960,8 +963,20 @@ While a leveraged system has no need to represent content here, its SSP must inc
...
+          <p>If 'not-applicable', attest explain why authentication is not applicable in the remarks.</p>
```
Comment (DimitriZhurkin) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:975]: Remove `attest`?
```suggestion
          <p>If 'not-applicable', explain why authentication is not applicable in the remarks.</p>
```

**Review by DimitriZhurkin — CHANGES_REQUESTED** [review]

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 4]
```diff
@@ -960,8 +963,20 @@ While a leveraged system has no need to represent content here, its SSP must inc
...
+        </remarks>
```
[the remark this answers is not in the document]
Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: I committed your suggestion: [b15dbc6](https://github.com/GSA/automate.fedramp.gov/pull/124/commits/b15dbc6db8f8bccd3e788805532bb04066d12181)

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md (hunk 2 above)
Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Thanks for the feedback. I removed these remarks. See: [b15dbc6](https://github.com/GSA/automate.fedramp.gov/pull/124/commits/b15dbc6db8f8bccd3e788805532bb04066d12181)

**Review by aj-stein-gsa — APPROVED** [review]

**Review by DimitriZhurkin — APPROVED** [review]

**Pull request merged**